calicoctl/commands/status.go

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) 2016 Tigera, Inc. All rights reserved.
```

Comment:
I _think_ we probably want this file to live in a subdirectory specific for the `node` command.

e.g `calicoctl/commands/node/status.go`

Keeps all the node options separated so they're easier to find, and makes it easier for us to add, for example, a `calicoctl status ...` command down the road if the need arises.